packages/training/deploy/docker/Dockerfile

```
@@ -0,0 +1,76 @@
+
+# Copy scripts
+COPY deploy/docker/scripts/ /app/scripts/
+RUN chmod +x /app/scripts/*.sh
+
+# Copy application code
+COPY python/ ./python/
+COPY Makefile ./Makefile
+COPY config/ ./config/
+
+# Install the training package
+RUN cd python && pip install -e .
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)